src/backend/app/Services/AbstractExportService.php

```diff
@@ -101,7 +102,8 @@ public function setActivatedSheet(string $sheetName): void {
         }
     }
 
-    public function saveSpreadSheet(): string {
+    public function saveSpreadSheet(?Filesystem $storage = null): string {
```

Comment:
🫣 I'm not a big fan... It feels like we are giving tooooo many configuration options which has little gain, but adds large complexity. 

What do you think about the following idea: 

- `AbstractExportService.php` is the only thing that's really using `disk('local')`
- Let's change it, to use default, cloud storage here as well

Basically, this gets rid of the distinction between `disk('default')` vs `disk('local')`. This has the additional benefit of making Local Dev and Prod more similar.